blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -431,7 +431,7 @@ private MetricData getBooleanMetricDataForRange(Locator locator, Range range, Gr
             }
         }
 
-        return new MetricData(points, getUnitString(locator), MetricData.Type.BOOLEAN);
+        return new MetricData(points, null, MetricData.Type.BOOLEAN);
```

Comment:
this seems like a bad code smell... if the unit isn't required, perhaps remove it from the constructor and have it optionally supplied via a method?